composer.json

```diff
@@ -36,7 +36,9 @@
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^3.2",
         "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/framework-bundle": "^3.2",
+        "alcaeus/mongo-php-adapter": "1.0.*",
```

Comment:
It's about php7 mongodb support. If you don't use this driver library, doctrine throws errors while installation about mongodb driver version. You can see the following stackOverflow post
http://stackoverflow.com/questions/36964171/php-7-with-doctrine-mongodb-odm